### PR TITLE
VM overview utilization card - donuts charts

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

## 📝 Description

First part of utilization card. this Is the utilization donuts chart and titles and summary. A follow-up pr will include more charts. 

## 🎥 Demo

image

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/utils/queris.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+export enum VMQueries {
```
Comment (glekner) [inline src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/utils/queris.ts:1]: change filename to `queries`

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/virtual-machines-overview-tab-utilization.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.VirtualMachinesOverviewTabUtilization--main {
+  .title {
```
Comment (glekner) [inline src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/virtual-machines-overview-tab-utilization.scss:2]: add margin bottom, or in another place.
add space between title and info

**Review by glekner — COMMENTED** [review]
looks good

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/virtual-machines-overview-tab-utilization.scss (hunk 2 above)
Comment (metalice) [reply src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/virtual-machines-overview-tab-utilization.scss:2]: you probably mean the subtitle , i spaced them :)

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/utils/queris.ts (hunk 1 above)
Comment (metalice) [reply src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/utils/queris.ts]: great catch!

**Pull request merged**